src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt

```diff
@@ -1123,9 +1165,41 @@ data class PatchDetail(
  */
 @Serializable
 data class Contributor(
+    @JsonProperty(
+        value = "org",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
     val org: String? = null,
+    @JsonProperty(
+        value = "name",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
     val name: String? = null,
+    @JsonProperty(
+        value = "email",
```

Comment:
What is email here, sorry?
I missed that field in timeline

Reply:
Ah, it’s contributor, why changed?

Reply:
it's a workaround for Jackson, Jackson doesn't support Kotlin data classes by default. So we need to ask our users to enable jackson library with KotlinModule or add such annotation for each fields in constructor (got getter and for field).